tests/trinity/conftest.py

```diff
@@ -51,6 +55,19 @@ def event_loop():
         loop.close()
 
 
+@pytest.fixture(scope='session')
```

Comment:
This seems potentially problematic to run at the session level due to leakage between tests?  Seems like it should be function scoped (also I'm surprised this works because I assumed the `event_loop` fixture was function scoped....

Reply:
Just to follow up on this ´function` scope works fine for the `event_bus` fixture but for the (existing) `event_loop` fixture it seems to need `session` scope or otherwise quite a bunch of tests fall apart. Not going to dig deeper as part of this PR.